src/components/Chart/constants.js

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import {
+  compose,
+  setDisplayName,
+  mapProps,
+  defaultProps,
+  setPropTypes,
+} from 'recompose';
+import C3Chart from 'react-c3js';
+
+import 'patternfly/dist/js/patternfly-settings';
+import 'patternfly/dist/js/patternfly-settings-charts';
+
+const patternfly = window.patternfly;
```

Comment:
really nitpick: I wonder if we should expose it via some other via, so in the future if this ever change, we don't need to change it here (and every other place that uses window.patternfly)

Reply:
good point 👍  . It seems like there is the concept of global constants common to all components now? Maybe some place in `src/common/constants.js` or `src/common/patternfly.js`?

Reply:
👍 moved to `src/common/patternfly.js` :-) and added to projects `index.js`